### PR TITLE
Create alt-text-scipy-2022.md

### PR DESCRIPTION
**Pull request by tonyfast**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Thank y'all for taking the time to prepare for the scavenger hunt, and wanting to improve your documentation.
+
+
+## How to participate.
+
+* Follow each link/code to a pull request.
+* Follow the pull request instructions to modify the respective alt text in [markdown] and [rst] formats.
+
+| JupyterLab    [^history]  | SciPy   | Dask | Bokeh |
```
Comment (gabalafou) [inline apps/labs/posts/alt-text-scipy-2022.md:41]: For some reason, the mdx compiler is failing on this table. Not sure why yet.

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+title: SciPy 2022 Alt Text Scavenger Hunt
+author: Tony Fast
+published: July 12, 2022
+description: 'Lorem ipsum dolor sit amet'
+category: [Accessbility, Open SOurce]
+# featuredImage:
```
Comment (gabalafou) [inline apps/labs/posts/alt-text-scipy-2022.md:7]: You can leave featured image out, but the page is not able to render correctly without a hero image.

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+---
+title: SciPy 2022 Alt Text Scavenger Hunt
+author: Tony Fast
+published: July 12, 2022
+description: 'Lorem ipsum dolor sit amet'
```
Comment (gabalafou) [inline apps/labs/posts/alt-text-scipy-2022.md:5]: I don't know what this is for but it shouldn't be lorem ipsum.

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+title: SciPy 2022 Alt Text Scavenger Hunt
+author: Tony Fast
```
Comment (gabalafou) [inline apps/labs/posts/alt-text-scipy-2022.md:3]: This needs to be `tony-fast`

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+## What next?
+
+* Start submitting and improving your alt text!
+    * [Working Jupyter alt text guide](https://github.com/Quansight-Labs/jupyter-accessibility-workshops/blob/main/docs/alt-text-guide.md#diagrams-charts-plots)
+    * [Chartability heuristics for accessible visualizations](https://chartability.fizz.studio/)
+* Stay tuned later this week to learn how you fight inaccessibility with our zine release.
+* Use more alt text in your social media and documentation.
+* Give the project maintainers time to merge your submissions.
+
+<!-- ![image](https://user-images.githubusercontent.com/4236275/176038647-78e2cb4a-5b17-4b15-8665-6b2990c68406.png) -->
```
Comment (gabalafou) [inline apps/labs/posts/alt-text-scipy-2022.md:60]: The mdx complier cannot handle HTML comments.

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: SciPy 2022 Alt Text Scavenger Hunt
-author: Tony Fast
+author: tony-fast
 published: July 12, 2022
 description: 'Lorem ipsum dolor sit amet'
```
Comment (tonyfast) [inline apps/labs/posts/alt-text-scipy-2022.md:5]:
```suggestion
description:"Announcing the SciPy 2022 Alt Text Scavenger Hunt"
```

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+[bokeh]: https://github.com/bokeh/bokeh/pull/12211
+[bokehqr]: https://user-images.githubusercontent.com/4236275/177208706-304e0176-969e-48a0-becd-730a8c92eb01.png
+[placeholder]: #
+[placeholderqr]: https://chart.googleapis.com/chart?cht=qr&chl=https%3A%2F%2Fgithub.com%2Falt-text-task-force&chs=300x300&choe=UTF-8&chld=L|2
+[^history]: The Alt Text workshop format started from Project Jupyter. The goal was a low-code, no-git workshop that worked for a lot of people. You can learn more about the history in our past blog posts, or even learn to run an event yourself
+[markdown]: #
+[rst]: #
+[tips]: #
+
+[^history]: These alt text events have been a community for mainstay for over a year.  In these [low-code workshops](https://labs.quansight.org/blog/2021/09/low-code-contributions-through-github) we have been able to remove the complexities of git in contributing to open source. These events have been run [in the past with several communities](https://github.com/isabela-pf/a11y-events/tree/main/workshop-resources/alt-text#past-events).
```
[the remark this answers is not in the document]
Comment (tonyfast) [reply apps/labs/posts/alt-text-scipy-2022.md]: yea they are part of the gfm standard. https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/

**Comment by bskinn** [comment]
Seem to be two things: (1) a malformed YAML header, and (2) `ipython` not recognized as a Code Hike language.

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+It is SciPy this week! It's one of the biggest community events in our corner of the world. We are so excited for the festivities, the talks, and see hear all our friends new ideas.
+
+Events, and technology, are increasingly more complex these days. This year SciPy is in-person and virtual, and we wanted to bring y'all two things that fit these constraints:
+
+1. We're introducing a pilot open source [scavenger hunt](#the-scavenger-hunt) where you can:
+   1. Contribute to popular open source projects.
+   2. Improve the alt text of open source documentation.
+2. A free digital _and limited print_ zine called [__How to Write Alt Text for Scientific Diagrams and Visualization__][zine] by [@marsbarlee](https://github.com/marsbarlee).
+
+## The scavenger hunt
```
Comment (bskinn) [inline apps/labs/posts/alt-text-scipy-2022.md:24]: If you're putting a period at the end of `## How to participate.`, you should also put one here.

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+[scipyqr]: https://user-images.githubusercontent.com/4236275/177211155-c9ef9a49-962d-429e-83ff-5ef1e9f96ca3.png
+[dask]: https://github.com/pavithraes/dask/pull/1
+[daskqr]: https://user-images.githubusercontent.com/4236275/177203555-1e0ef684-8113-4150-8909-2996cfbe060e.png
+[bokeh]: https://github.com/bokeh/bokeh/pull/12211
+[bokehqr]: https://user-images.githubusercontent.com/4236275/177208706-304e0176-969e-48a0-becd-730a8c92eb01.png
+[placeholder]: #
+[placeholderqr]: https://chart.googleapis.com/chart?cht=qr&chl=https%3A%2F%2Fgithub.com%2Falt-text-task-force&chs=300x300&choe=UTF-8&chld=L|2
+[zine]: https://heyzine.com/flip-book/f47006ad71.html
+
+[^1]: These alt text events, that started in Project Jupyter, have been a community mainstay for over a year. In these [low-code workshops](https://labs.quansight.org/blog/2021/09/low-code-contributions-through-github) we have been able to remove the complexities of git in contributing to open source. These events have been run [in the past with several communities](https://github.com/isabela-pf/a11y-events/tree/main/workshop-resources/alt-text#past-events).
```
Comment (bskinn) [inline apps/labs/posts/alt-text-scipy-2022.md:72]: The "and thus expand participation" is optional -- just think the thought is ~incomplete without it.

```suggestion
[^1]: These alt text events, which started in Project Jupyter, have been a community mainstay for over a year. In these [low-code workshops](https://labs.quansight.org/blog/2021/09/low-code-contributions-through-github) we have been able to remove the complexities of git in contributing to open source and thus expand participation. These events have been run [in the past with several communities](https://github.com/isabela-pf/a11y-events/tree/main/workshop-resources/alt-text#past-events).
```

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+2. SciPy
+3. Dask
+4. Bokeh
+
+Thank y'all for taking the time to prepare for the scavenger hunt, and wanting to improve your documentation.
+
+## How to participate.
+
+- Follow each link/code to a pull request.
+- Follow the pull request instructions to modify the respective alt text in markdown and rst formats.
```
Comment (bskinn) [inline apps/labs/posts/alt-text-scipy-2022.md:38]:
```suggestion
- Follow the pull request instructions to modify the respective alt text in markdown or rst format, as appropriate.
```

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+
+| JupyterLab [^1]     | SciPy                 | Dask                | Bokeh                 |
+| ------------------- | --------------------- | ------------------- | --------------------- |
+| [![][jlabqr]][jlab] | [![][scipyqr]][scipy] | [![][daskqr]][dask] | [![][bokehqr]][bokeh] |
+
+## Why participate?
+
+- _low barrier to entry_ to participate in open source projects
+- _improve the accessibility of open source documentation_
+- _learn about more alt text_
```
Comment (bskinn) [inline apps/labs/posts/alt-text-scipy-2022.md:48]: I think?

```suggestion
- _learn more about alt text_
```

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
+---
+title: SciPy 2022 Accessibility Awareness Programs
+author: tony-fast
+published: July 12, 2022
+description: "Announcing the SciPy 2022 Accessibility Awareness Efforts"
+category: [Accessibility]
+# featuredImage:
+#   src: /posts/hello-world-post/featured.png
+#   alt: 'Excellent alt-text describing the featured image'
```
Comment (bskinn) [inline apps/labs/posts/alt-text-scipy-2022.md:9]: Probably want to actually have alt text here?

First pass: "Close crop of a cartoon page showing a person writing alt text on an image that is part of a map, along with portions of text reading "ALT TEXT ... SCIENTIFIC DIAGRAMS ... DOCUMENTATION"

File: apps/labs/posts/alt-text-scipy-2022.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
+---
+title: SciPy 2022 Accessibility Awareness Programs
+author: tony-fast
+published: July 12, 2022
```
Comment (bskinn) [inline apps/labs/posts/alt-text-scipy-2022.md:4]: This'll actually be on the 13th?

**Review by bskinn — COMMENTED** [review]
Few suggestions on a quick pass.

File: apps/labs/posts/alt-text-scipy-2022.md (hunk 12 above)
Comment (tonyfast) [reply apps/labs/posts/alt-text-scipy-2022.md:9]: these are commented out. there's alt text on the hero image. i didnt include a feature image.

**Comment by tonyfast** [comment]
thanks for the :eyes: @bskinn . i addressed all your review comments.

**Pull request merged**